.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+bower_components
+coverage
+public
+reverse-proxy
```

Comment:
can you remove these? There is no bower_components, coverage, public, or reverse-proxy in this repository